resources/assets/js/models/types.ts

```diff
@@ -248,3 +248,37 @@ export interface JobPosterKeyTask {
     description: string;
   };
 }
+
+interface Link {
+  url: string | null;
+  text: string;
+  title: string;
+}
+
+interface Activity {
+  count: number;
+  new: Link;
+}
+
+export interface JobAction {
+  activity: Activity;
+  applicants: number;
+  classification: string;
+  draft: Link;
+  managerTime: number;
+  owned: boolean;
+  preview: Link;
+  screeningPlan: Link;
+  summary: Link;
+  status: JobStatus;
+  title: string;
+  userTime: number;
+}
+
+export interface UnclaimedJob {
+  title: string;
+  createdAt: string;
+  status: JobStatus;
+  hiringManagers: string[];
+  hrAdvisors: string[];
+}
```

Comment:
This file is meant to contain Domain types, the most basic representations of the data the app uses. These interfaces look like they're pretty specific to individual components. They should just be exported from those component files, if necessary.

Activity, and things like managerTime/userTime, will eventually need domain types, but I think we're still a bit too unclear on how they should work to add them yet. 😅